packages/semaphore/README.md

```diff
@@ -0,0 +1,64 @@
+# `@shopify/semaphore`
+
+[![Build Status](https://travis-ci.org/Shopify/quilt.svg?branch=master)](https://travis-ci.org/Shopify/quilt)
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](LICENSE.md) [![npm version](https://badge.fury.io/js/%40shopify%2Fsemaphore.svg)](https://badge.fury.io/js/%40shopify%2Fsemaphore.svg)  [![npm bundle size (minified + gzip)](https://img.shields.io/bundlephobia/minzip/@shopify/semaphore.svg)](https://img.shields.io/bundlephobia/minzip/@shopify/semaphore.svg)
+
+Counting [semaphore](https://en.wikipedia.org/wiki/Semaphore_(programming))
```

Comment:
This explanation is just a little too generic.
If you can add more explaining on what this package does.
Specially things that can answer `In x situation I would want to use this package` type question.